### ISSUE TITLE
Add background = "box" as a more-intuitive theme_grattan option

### ISSUE THREAD

**Issue by wfmackey** [MattCowgill/grattantheme#14]
Mabe: 
Add a `theme_grattan(background = "box")` option which does the same thing as the current `theme_grattan(background = "orange")`.

Thought came from the phrasing of the vignette: 
> Sometimes we'll want a chart for a box in a report. We can change the background colour with the `background` argument.

For which `"box"` is more intuitive. @MattCowgill any objections?

**Comment by MattCowgill** [comment]
Go for it, but can you leave it so orange also works?

**Comment by wfmackey** [comment]
Of course -- will just change to:

```
...
if (background == "orange" |  background == "box") {
    ret <- ret + ggplot2::theme(rect = ggplot2::element_rect(fill = grattantheme::grattan_orange_alpha))
  }
```

**Comment by MattCowgill** [comment]
I see you've done this in the `vignettes-plus` branch now, so I'll close this

**Comment by wfmackey** [comment]
Nah nah -- all issues are stored here. You create a new branch when you want to make significant changes, but it doesn't create a new repository (that's 'forking'). When you're happy with the changes etc, in your branch you create a 'pull request' to merge the branch with the 'master' branch. AND THEN you can reference the completed pull request like you do an issue, and I could say "#15 closes this issue" and this issue will be closed.

God that sounds like gibberish but I hope you make sense of it.

**Comment by wfmackey** [comment]
tldr: When I merge my `vignette-plus` branch, which has the code that closes this issue, I'll close the issue.

**Comment by MattCowgill** [comment]
yeah I know all that (branches, forking), just figured there was no point leaving the issue open given that you've written the code, but fair enough if you want to leave it until you're done

**Comment by wfmackey** [comment]
Ah okay yep. I think best to keep it open until the fix has been added to the master code?

**Comment by wfmackey** [comment]
Fixed in #17
